frontend/src/routers/ProjectRouter/ApplicationRouter.tsx

```diff
@@ -29,13 +28,7 @@ const HitTargets = React.lazy(() => import('../../pages/Buttons/HitTargets'))
 const ApplicationRouter: React.FC = () => {
 	const { projectId } = useNumericProjectId()
 	const { page, searchQuery } = useSearchContext()
-	const { page: errorPage, searchQuery: errorSearchQuery } =
-		useErrorSearchContext()
 	usePreloadSessions({ page: page || 1, query: JSON.parse(searchQuery) })
-	usePreloadErrors({
-		page: errorPage || 1,
-		query: JSON.parse(errorSearchQuery),
-	})
```

Comment:
Just curious, does preloading not work with the new way we are fetching errors, or is there another reason we wanted to get rid of this?

Reply:
I believe this was disabled by default, so believe we were going getting rid of this